### PR TITLE
feat(widget-table): add features(rapidIncrease, link, icon) & apply table style

### PR DESCRIPTION
**Pull request by sulmoJ**
### To Reviewers
- [ ] Skip (`style`, `chore` ONLY)
- [ ] Not that difficult

### Type of Change
- [ ] New feature
- [ ] Bug fixes
- [ ] Feature improvement
- [ ] Refactor
- [ ] Others (performance improvement, refactoring, CI/CD, etc.)

### Checklist
- [ ] `Error / Warning / Lint / Type`

### Description
image

- New features
    - add link option
    - add rapid increase option
    - add icon option
    - table style
- Unimplemented features
    - pagination
    - currency
    - detail popover

### CODE REVIEW

File: src/services/dashboards/widgets/_components/WidgetDataTable.vue [hunk 1]
```diff
@@ -80,11 +82,30 @@
                                                       @click.stop="handleClickLegend(rowIndex)"
                                             />
                                         </template>
-
+                                        <template v-if="field?.icon">
+                                            <p-i :name="getHandler(field.icon, item)"
+                                                 :width="'1rem'"
+                                                 :height="'1rem'"
```
Comment (minseolee) [inline src/services/dashboards/widgets/_components/WidgetDataTable.vue:88]:
```suggestion
                                                 width="1rem"
                                                 height="1rem"
```
How about this?

File: src/services/dashboards/widgets/_components/WidgetDataTable.vue [hunk 2]
```diff
@@ -239,6 +267,13 @@ export default defineComponent<Props>({
             }
             return item;
         };
+        const getHandler = (option: Field['icon']|Field['link']|Field['rapidIncrease'], item) => {
```
Comment (minseolee) [inline src/services/dashboards/widgets/_components/WidgetDataTable.vue:270]: This handler's return type looks like little bit confusing. How about adding return type?

**Review by minseolee — APPROVED** [review]
LGTM. left small comments :)

File: src/services/dashboards/widgets/_components/WidgetDataTable.vue [hunk 3]
```diff
@@ -239,6 +267,13 @@ export default defineComponent<Props>({
...
+            if (typeof option === 'string' || typeof option === 'boolean') {
+                return option;
+            }
+            if (option) return option(item);
+            return undefined;
```
Comment (WANZARGEN) [inline src/services/dashboards/widgets/_components/WidgetDataTable.vue:275]: praise: 👍

**Review by WANZARGEN — APPROVED** [review]
LGTM!! getHandler is great idea :)

**Pull request merged**